js/views/file_input_view.js

```diff
@@ -104,6 +104,8 @@
                     do {
                         i = i - 1;
                         // dataURLtoBlob -> components/blueimp-canvas-to-blob
+                        // TODO: Replace with native `Canvas::toBlob`:
+                        // https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement/toBlob
```

Comment:
Now that we have a known browser target, we could leverage the standard `Canvas::toBlob` API, although it is async.